uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
```

Comment:
That looks fine, actually.  Remember that the way the code is written now, we default to just using ``group.new_ribbon`` which is the same as what it did before.

Looking at the above three cases you listed (I think I remember what all of those cases are):
- The first is when we create the group in the first place; we're not passing a ribbon, so it'll just use the default like before.  So that's fine.
- The second is where someone clicks the "This person isn't coming" button on the group page, which deletes the badge and then creates a new one to replace it.  Previously the new badge would just have the group default ribbon, which is what happens now.  So this is fine.
- The third is where groups add their own badges.  In this case, we want the ribbons to be of the group default type, which is both what the old code did and what the new code does.

So this looks okay and seems good to merge.